src/main/java/com/customer/controller/JwtAuthenticationController.java

```diff
@@ -0,0 +1,51 @@
+package com.customer.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.customer.exception.ForbiddenException;
+import com.customer.jwtconfig.JwtTokenUtil;
+import com.customer.model.Customer;
+import com.customer.model.JwtResponse;
+import com.customer.service.JwtUserDetailsService;
+
+import lombok.extern.slf4j.Slf4j;
+
+@RestController
+/*
+ * For supporting CORS @CrossOrigin is supported by SpringBoot and by
+ * default @CrossOrigin allows all origin and HTTP methods specified
+ */
+@CrossOrigin
+@Slf4j
+public class JwtAuthenticationController {
+
+	@Autowired
+	private AuthenticationManager authenticationManager;
+
+	@Autowired
+	private JwtTokenUtil jwtTokenUtil;
+
+	@Autowired
+	private JwtUserDetailsService userDetailsService;
+
+	@PostMapping(value = "/authenticate")
```

Comment:
if you are using `@PostMapping` there is no need for specifying `value`

`@PostMapping("/authenticate")` would do

Reply:
Done.....